src/main/java/studio/magemonkey/fabled/data/sql/SQLManager.java

```diff
@@ -0,0 +1,80 @@
+package studio.magemonkey.fabled.data.sql;
+
+import lombok.Getter;
+import studio.magemonkey.fabled.Fabled;
+import studio.magemonkey.fabled.data.sql.tables.FabledPlayersSQL;
+
+import java.io.File;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class SQLManager {
+
+    @Getter
+    private static volatile Connection connection;
+
+    private static FabledPlayersSQL fabledPlayersSQL;
+
+    public static void init() {
+        String host = Fabled.getSettings().getSqlHost();
+        int port = Integer.parseInt(Fabled.getSettings().getSqlPort());
+        String database = Fabled.getSettings().getSqlDatabase();
+        String user = Fabled.getSettings().getSqlUser();
+        String password = Fabled.getSettings().getSqlPass();
+
+        Fabled.inst().getLogger().info("Initializing SQLManager with type: MySQL");
+        connection = getMySQLConnection(host, port, database, user, password);
+
+        if (connection == null) {
+            Fabled.inst().getLogger().severe("Failed to initialize the Connection.");
+        }
+
+        fabledPlayersSQL = new FabledPlayersSQL();
+        fabledPlayersSQL.migrateTable();
```

Comment:
We only really need to run this migration on the _first_ run. Because `connection()` calls init whenever the connection is closed, this migration will be run time and time again. It's also creating a new FabledPlayersSQL object every time. We can probably store the database info in some private class variables and eliminate the need to run the entire init method just to establish a new connection to the DB.

We can also add a `reload` method to refresh the database info from the settings and at that point, I think attempting a migration again would be a decent idea in case the database credentials have changed.

Reply:
Reworked with the next commit too!